one/src/lib.rs

```diff
@@ -245,28 +274,33 @@ impl Daemon {
         };
         debug!("using directory: {}", dir.display());
 
-        let mut p2p_config = Libp2pConfig::default();
-        p2p_config.mdns = opts.mdns;
-        p2p_config.bitswap_server = true;
-        p2p_config.bitswap_client = true;
-        p2p_config.kademlia = true;
-        p2p_config.autonat = true;
-        p2p_config.relay_server = true;
-        p2p_config.relay_client = true;
-        p2p_config.gossipsub = true;
-        p2p_config.max_conns_out = 2000;
-        p2p_config.max_conns_in = 2000;
-        p2p_config.bootstrap_peers = opts
-            .bootstrap_addresses
-            .iter()
-            .map(|addr| addr.parse())
-            .collect::<Result<Vec<Multiaddr>, multiaddr::Error>>()?;
-
-        p2p_config.listening_multiaddrs = opts
-            .swarm_addresses
-            .iter()
-            .map(|addr| addr.parse())
-            .collect::<Result<Vec<Multiaddr>, multiaddr::Error>>()?;
+        let p2p_config = Libp2pConfig {
+            mdns: opts.mdns,
+            bitswap_server: true,
+            bitswap_client: true,
+            kademlia: true,
+            autonat: true,
+            relay_server: true,
+            relay_client: true,
+            gossipsub: true,
+            max_conns_out: opts.max_conns_out,
+            max_conns_in: opts.max_conns_in,
+            max_conns_pending_out: opts.max_conns_pending_out,
```

Comment:
looks like all these opts other than `max_conns_in` and `max_conns_out` weren't being set by us before.  Do you know what they defaulted to?  Are we expecting setting this now with the defaults in this PR to change any behavior, or did we match the defaults here to the defaults within libp2p?

Reply:
I pulled all the existing defaults up to this layer of the code. So no behavior change is expected. FWIW you can find the previous defaults here https://github.com/3box/rust-ceramic/blob/main/p2p/src/config.rs#L91-L95